sdk/storage/Azure.Storage.Blobs/src/autorest.md

```diff
@@ -4,7 +4,7 @@ Run `dotnet build /t:GenerateCode` to generate code.
 
 ``` yaml
 input-file:
-    - https://raw.githubusercontent.com/Azure/azure-rest-api-specs/f6f50c6388fd5836fa142384641b8353a99874ef/specification/storage/data-plane/Microsoft.BlobStorage/stable/2024-08-04/blob.json
+    - C:\azure-rest-api-specs\specification\storage\data-plane\Microsoft.BlobStorage\stable\2025-01-05\blob.json
```

Comment:
TODO fix this when https://github.com/Azure/azure-rest-api-specs/pull/30649 merges